src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1104,6 +1104,9 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                         if ($_.Exception -and $_.Exception.WasThrownFromThrowStatement) {
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-windows`
Tests for $ErrorView.ConciseView tests.Activity shows up correctly for scriptblocks
```powershell
Expected like wildcard '*Write-Error: *myError*' to match $null, but it did not match.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Formatting\ErrorView.Tests.ps1: line 53
53:             $e[2] | Should -BeLike "*Write-Error: *myError*" # wildcard due to VT100
```
Stream writer tests.Redirect Stream Tests.Should write error messages to the error stream
```powershell
Expected ': Testing Error' to be found in file 'C:\Users\VssAdministrator\AppData\Local\Temp\db0f0026-acd8-466d-9fba-a6c8dd29b4a5\writeoutput.txt', but it was not found.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Write-Stream.Tests.ps1: line 33
33:             $targetfile | Should -FileContentMatch ": Testing Error"
```